perf/benchmarkapps/Shared/Certs/Readme.md

```diff
@@ -0,0 +1 @@
+These certs are generated via openssl according to https://stackoverflow.com/questions/37714558/how-to-enable-server-side-ssl-for-grpc. The server.crt and server.key were combined into server.pfx. The password is 1111. These certs are not secure, do not use in production.
```

Comment:
IIRC we were thinking of unifying the test certs being used with the ones from grpc/grpc (currently used for interop tests), is that still the plan or we just need a different set of certs here because of infrastructure reasons (e.g. needing to share the same certs with the rest of Asp.net core benchmarking infrastructure)?

Reply:
It's planned, but low priority.